addons/knobs/README.md

```diff
@@ -264,6 +264,23 @@ const value = select(label, options, defaultValue, groupId);
 
 > You can also provide options as an array like this: `['red', 'blue', 'yellow']`
 
+
+### radio buttons
+
+Allows you to get a value from a list of radio buttons from the user.
+
+```js
+import { radioButtons } from '@storybook/addon-knobs';
+
+const options = {
+      Kiwi: 'kiwi',
+      Guava: 'guava',
+      Watermelon: 'watermelon',
+    };
+
+const  = radioButtons(label, options, defaultValue, groupId);
```

Comment:
This example is not copy-paste-able since you don't have `label`, `defaultValue` and `groupId`.

Reply:
I fixed the parameters for radioButtons.